setup.py

```diff
@@ -23,7 +23,8 @@
     install_requires=[
         'futures',
         'protobuf>=3.0.0b2.post2',
-        'opentracing>=1.2.1,<1.3'
+        'opentracing>=1.2.1,<1.3',
+        'six>=1.10.0',
```

Comment:
Oh I see. >= 1.10 would work for you? Otherwise, we can simply go with < 2.0, as you have suggested ;)